classroom/snippets/src/main/java/ListStudentSubmissions.java

```diff
@@ -0,0 +1,90 @@
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+// [START classroom_list_student_submissions_class]
+
+import com.google.api.client.googleapis.json.GoogleJsonError;
+import com.google.api.client.googleapis.json.GoogleJsonResponseException;
+import com.google.api.client.http.HttpRequestInitializer;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.gson.GsonFactory;
+import com.google.api.services.classroom.Classroom;
+import com.google.api.services.classroom.ClassroomScopes;
+import com.google.api.services.classroom.model.ListStudentSubmissionsResponse;
+import com.google.api.services.classroom.model.StudentSubmission;
+import com.google.auth.http.HttpCredentialsAdapter;
+import com.google.auth.oauth2.GoogleCredentials;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/* Class to demonstrate the use of Classroom List StudentSubmissions API. */
+public class ListStudentSubmissions {
+  /**
+   * Lists student submissions based on courseId, courseWorkId, and userId.
+   *
+   * @param courseId - identifier of the course.
+   * @param courseWorkId - identifier of the course work.
+   * @param userId - identifier of the student whose work to return.
+   * @return - list of student submissions.
+   * @throws IOException - if credentials file not found.
+   */
+  public static List<StudentSubmission> listStudentSubmissions(String courseId, String courseWorkId,
+      String userId) throws IOException {
+    /* Load pre-authorized user credentials from the environment.
+     TODO(developer) - See https://developers.google.com/identity for
+      guides on implementing OAuth2 for your application. */
+    GoogleCredentials credentials = GoogleCredentials.getApplicationDefault()
+        .createScoped(Collections.singleton(ClassroomScopes.CLASSROOM_COURSEWORK_STUDENTS));
+    HttpRequestInitializer requestInitializer = new HttpCredentialsAdapter(
+        credentials);
+
+    // Create the classroom API client.
+    Classroom service = new Classroom.Builder(new NetHttpTransport(),
+        GsonFactory.getDefaultInstance(),
+        requestInitializer)
+        .setApplicationName("Classroom samples")
+        .build();
+
+    // [START classroom_list_student_submissions_code_snippet]
+
+    List<StudentSubmission> studentSubmissions = new ArrayList<>();
+    try {
+      // Set the userId as a query parameter on the request.
+      ListStudentSubmissionsResponse response = service.courses().courseWork().studentSubmissions()
```

Comment:
I understand that we can't create submissions through the API, but just as a general question, were you able to test these calls with manually-created submissions?